api/trace/current.go

```diff
@@ -25,10 +25,16 @@ var (
 )
 
 func SetCurrentSpan(ctx context.Context, span Span) context.Context {
+	if ctx == nil {
```

Comment:
why these changes?

Reply:
In my exporter, I want to know the current span in order to annotate the span data with accumulated metrics, grouped by span_id. I want this to happen for all callers, without the programmer having to do anything. So this check avoids a crash when you ask for the current span and there's no context.